### PR TITLE
Support {align}, {align*}, {alignat}, and {alignat*}

### PR DESCRIPTION
**Pull request by ronkok**
Now that PR #2183 is merged, the next four AMS environments are pretty straightforward.

### CODE REVIEW

**Comment by ronkok** [comment]
CircleCI is misbehaving and not producing screenshotter artifacts. So I do not have updated screenshots. I'll try again tomorrow. If any one can suggest a course of action, I'll try that too.

**Comment by ylemkimon** [comment]
Since new screenshot tests require display mode, it should be set in the screenshotter. See `ArrayMode` screenshot for example.

**Comment by ronkok** [comment]
Well, there I have a screenshot working. @ylemkimon Thank you for the tip.

I think this PR is now ready for review.

File: test/screenshotter/ss_data.yaml [hunk 1]
```diff
@@ -32,7 +35,7 @@ Alignedat: |
         a &= 1 & b &= 2 &\quad c &= 3\\
         3a &= 3 & 17b &= 34 &\quad 400c &= 1200
     \end{alignedat}
-Arrays: |
+Arrays:
```
Comment (ylemkimon) [inline test/screenshotter/ss_data.yaml:38]:
```suggestion
Arrays: |
```

**Comment by ronkok** [comment]
@ylemkimon Thank you for the review!

**Pull request merged**